### DISCUSSION TITLE
postUpgradeTasks trouble

### DISCUSSION THREAD

**Discussion by UberMouse** [renovatebot/renovate#8820] [Request Help] [answered]
I'm trying to get Renovate working with RushJS based on the info I found in #3681 but my postUpgradeTasks aren't running and I can't find anything in the debug logs about it.

Here is my full config. The goal I'm trying to accomplish is to get renovate to invoke the rush install/run script to run `rush update` after the package.json files have been modified and then include any changed files under the root `common` directory in the commit.

But I continue to see the existing behaviour of renovate updating the package.json files and including nothing else in the commit. Example job run https://app.renovatebot.com/dashboard#github/koordinates/kawaka/306441308

```
{
  "extends": [
    "config:base"
  ],
  "packageRules": [
    {
      "depTypeList": ["devDependencies"],
      "automerge": true
    },
    {
      "depTypeList": ["peerDependencies"],
      "rangeStrategy": "widen"
    },
    {
      "description": "jest monorepo",
      "sourceUrlPrefixes": [
        "https://github.com/facebook/jest"
      ],
      "packagePatterns": [
        "^@types/jest",
        "^ts-jest"
      ]
    }
  ],
  "separateMinorPatch": true,
  "patch": {
    "enabled": false
  },
  "masterIssue": true,
  "rangeStrategy": "pin",
  "semanticCommits": true,
  "stabilityDays": 3,
  "prCreation": "not-pending",
  "ignorePaths": ["packages/react-redux-grid", "packages/graphql-transport-electron"],
  "ignoreDeps": ["webdriverio", "flowbin"],
  "rebaseWhen": "never",
  "schedule": ["after 7am and before 7pm"],
  "postUpgradeTasks": {
    "commands": ["node common/scripts/install-run-rush.js update"],
    "fileFilters": ["common/**/*"]
  }
}
```

**Comment by viceice** [comment, marked as answer]
`postUpgradeTasks` are only available on self hosted renovate.

**Reply by UberMouse** [reply]
So it's currently impossible to use Renovate with RushJS monorepos until Renovate adds native support, unless you self host? Or is there some other workaround.

**Reply by rarkins** [reply]
You could potentially add a GitHub Action to force push a `rush update` commit on top of Renovate's branches
